GLTFSDK/Inc/GLTFSDK/ExtensionsKHR.h

```diff
@@ -70,6 +70,28 @@ namespace Microsoft
                 std::string SerializeDracoMeshCompression(const DracoMeshCompression& dracoMeshCompression, const Document& gltfDocument);
                 std::unique_ptr<Extension> DeserializeDracoMeshCompression(const std::string& json);
             }
+
+            namespace TextureInfos
```

Comment:
The original idea was that the namespace would be the first, 'common', part of the extensions name (e.g. 'materials', 'lights' or 'techniques') but annoyingly the draco extension (and to a lesser extent the texture transform extension) don't follow the same naming convention. Rather than use 'TextureInfos', which looks a little strange, I'd go with 'Texture' or 'Textures'

Reply:
The thing is, these are definitely not related to Textures (as in glTF::Texture), they are related to glTF::Material::TextureInfo.

Reply:
That was my thinking too. Couple options: remove the plurals (plurality?) from all three namespaces or rename them from e.g. TextureInfos to TextureInfoExtensions?

Reply:
Well I'm not convinced the addition of 'Extensions' is much better and it is now a breaking change. Probably better to just go back to what you had before